bitcoin-review-podcast/lightning-privacy-splice-panel.md

```diff
@@ -413,32 +425,32 @@ But I think there are things that we can explore here that can make it potential
 
 ## Gossip filters
 
-Speaker 1: 00:24:20
+Jeff: 00:24:20
 
 Would that mobile client then, in order to do this sort of local sync of the gossip, which is maybe, I guess, it's preferable for a mobile user, Gossip is pretty expensive as far as network bandwidth and downloading it all.
 Does that require then like a sort of custom implementation for the mobile wall to do that sort of local sync?
 Now in the LDK side we have like this rapid gossip sync which makes it pretty quick.
 
-Speaker 6: 00:24:46
+Bastien Teinturier: 00:24:46
 
 And the other idea is in the very old trampoline PRs I had like three years ago, there was a proposal to do gossip filters that would be applied that you ask your peers to apply some gossip filters so that they forward to you only a subset of the gossip that interests you, so that mobile wallets can get a fraction of the gossip that they're interested in without wasting too much bandwidth.
 But this is still waiting for people to play with Trampoline before we do something like that.
 And this will have to be reevaluated to see if it really makes sense compared to something like rapid gossip syncs or other ways of syncing subparts of the graph.
 But I think that there's a large design space here to do it in a lot of different ways that could help
 
-Speaker 4: 00:25:30
+Dusty Damon: 00:25:30
 
 your practice.
```

Comment:
There is a broken sentence here. You should make sure that the speaker attributions are correct as you are listening to the original source. Please check that in the rest of the transcript as I already commented on 2 of those occasions.